extendr-api/src/scalar/rbool.rs

```diff
@@ -14,12 +14,25 @@ pub struct Rbool(i32);
 impl Rbool {
     gen_impl!(Rbool, i32);
 
-    pub fn t() -> Rbool {
+    pub const fn t() -> Rbool {
         Rbool(1)
     }
-    pub fn f() -> Rbool {
+
+    pub const fn f() -> Rbool {
         Rbool(0)
     }
+
+    pub const fn na_const() -> Rbool {
```

Comment:
But we have it as a part of `CanBeNA` trait? What's the difference?

Reply:
Surprisingly, trait methods can't be const!

Reply:
I'm working hard to avoid `Rbool(pub i32)`

Reply:
You could take a peek at:

https://docs.rs/num-complex/latest/num_complex/struct.Complex.html

We could have this on a feature.